### PR TITLE
Initial commit of the PHP server quickstart

### PR DESCRIPTION
**Pull request by bobiechen-twilio**
### Validation
- Validated endpoint functions with [Swift Voice Quickstart](https://github.com/twilio/voice-quickstart-swift).
- Validated endpoint functions with python tests

### CODE REVIEW

File: accessToken.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+/*
+ * Creates an access token with VoiceGrant using your Twilio credentials.
+ */
+include('./vendor/autoload.php');
+include('./config.php');
+use Twilio\Jwt\AccessToken;
+use Twilio\Jwt\Grants\VoiceGrant;
+
+// Use identity and room from query string if provided
+$identity = isset($_GET["identity"]) ? $_GET["identity"] : "identity";
```
Comment (bobiechen-twilio) [inline accessToken.php]: By default PHP supports both `GET` and `POST` without any specifier in the source. We are assuming the request is coming in as `GET` but we want all the endpoints to support both methods.

File: placeCall.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+include('./vendor/autoload.php');
+include('./config.php');
+
+$callerId = 'client:quick_start';
+$to = isset($_GET["to"]) ? $_GET["to"] : "alice";
+
+$client = new Twilio\Rest\Client($API_KEY, $API_KEY_SECRET, $ACCOUNT_SID);
+
+// Read TwiML at this URL when a call connects (hold music)
+$call = $client->calls->create(
```
Comment (idelgado) [inline placeCall.php]: Some functionality is missing. See Python PR.

File: makeCall.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+/*
+ * Use a valid Twilio number by adding to your account via https://www.twilio.com/console/phone-numbers/verified
+ */
+$callerNumber = '1234567890';
+
+$response = new Twilio\Twiml();
+if (empty($to) || !isset($to)) {
+  $response->say('Congratulations! You have just made your first call! Good bye.');
+} else if (is_numeric($to)) {
```
Comment (idelgado) [inline makeCall.php]: You need to replace `+` character with `` and then check for is_numeric unless `is_numeric` ignores `+` characters. I'll add a test for this.

**Review by idelgado — CHANGES_REQUESTED** [review]

File: makeCall.php [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+ * In order to make an outgoing call using Twilio Voice SDK, you need to provide a
+ * TwiML App SID in the Access Token. You can run your server, make it publicly
+ * accessible and use `/makeCall` endpoint as the Voice Request Url in your TwiML App.
+ */
+include('./vendor/autoload.php');
+include('./config.php');
+
+$callerId = 'client:quick_start';
+$to = isset($_POST["to"]) ? $_POST["to"] : NULL;
+if (empty($to) || !isset($to)) {
```
Comment (idelgado) [inline makeCall.php]: Usually you want to check for isset before you check for empty. Is that true for PHP?

File: makeCall.php [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  $to = isset($_GET["to"]) ? $_GET["to"] : NULL;
+}
+
+/*
+ * Use a valid Twilio number by adding to your account via https://www.twilio.com/console/phone-numbers/verified
+ */
+$callerNumber = '1234567890';
+
+$response = new Twilio\Twiml();
+if (empty($to) || !isset($to)) {
```
Comment (idelgado) [inline makeCall.php]: Same here.

File: makeCall.php [hunk 6]
```diff
@@ -23,7 +23,7 @@
 $response = new Twilio\Twiml();
 if (empty($to) || !isset($to)) {
   $response->say('Congratulations! You have just made your first call! Good bye.');
-} else if (is_numeric($to)) {
+} else if (is_numeric(str_replace('', '+', $to))) {
```
Comment (idelgado) [inline makeCall.php]: should be `str_replace('+', '', $to)`

File: placeCall.php [hunk 7]
```diff
@@ -5,18 +5,38 @@
...
+  $call = $client->calls->create(
+    'client:alice', // Call this number
+    $callerId,      // From a valid Twilio number
+    array(
       'url' => 'https://'.$_SERVER['HTTP_HOST'].'/incoming.php'
-  )
-);
+    )
+  );
+} else if (is_numeric(str_replace('', '+', $to))) {
```
Comment (idelgado) [inline placeCall.php]: Same here.

Comment (idelgado) [reply placeCall.php]: http://php.net/manual/en/function.str-replace.php

**Review by idelgado — CHANGES_REQUESTED** [review]

File: placeCall.php [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+/*
+ * Makes a call to the specified client using the Twilio REST API.
+ */
+include('./vendor/autoload.php');
+include('./config.php');
+
+$identity = 'alice';
+$callerNumber = '1234567890';
+$callerId = 'client:quick_start';
+$to = isset($_GET["to"]) ? $_GET["to"] : NULL;
```
Comment (idelgado) [inline placeCall.php]: Support for POST is missing

**Review by idelgado — CHANGES_REQUESTED** [review]

**Review by idelgado — APPROVED** [review]

**Pull request merged**